src/.php_cs

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
What the role of this file?

Reply:
This should be removed, I think semantic-release script couldn't run generate.sh file